README.md

```diff
@@ -6,14 +6,38 @@
 
 # caMicroscope distribution
 
-
 run with `docker-compose -f caMicroscope.yml up`
 
 this will build all services and run in the foreground.
 Use `docker-compose -f caMicroscope.yml build` to rebuild the services.
 
 Once everything is up, go to <the host this is running on>:4010/ to see the landing page.
 
+To use docker-compose and run the application, install Docker Desktop Application. <br>
+**NOTE for Windows users:** The Docker Desktop Application also need WSL2 so follow the setup for Windows given below to install Docker on Windows 10. 
+
+**Setup for Windows:**
+1. Install Windows Subsystem for Linux (WSL2) from the link given here: <a href = "https://docs.microsoft.com/en-us/windows/wsl/install-win10#manual-installation-steps">WSL2</a>, **FOLLOW STEPS 1 to 5 ONLY (Step 6 - Install your Linux distribution of choice IS NOT REQUIRED)** 
+2. After completing all the five steps, <a href = "https://hub.docker.com/editions/community/docker-ce-desktop-windows/">Install Docker Desktop for Windows</a>, click on the "Get Docker" button on the right and run the "Docker Desktop Installer" to complete the installation.
+
+# Fast Local Changes
+When using the hosted setup, you can have the distribution host the changes from your local. Follow these steps :<br>
+1. Clone <a href = "https://github.com/camicroscope/Distro">Distro</a> repository, the <a href = "https://github.com/camicroscope/Caracal">Caracal </a> repository and the <a href = "https://github.com/camicroscope/caMicroscope">caMicroscope </a>repository in the same parent directory.<br>
+2. Set the build to build your local changes instead of the hosted git versions by editing the ca-back container section of your **'develop.yml' of Distro repository**. 
+3. Replace the build context section with the path to your local caracal folder("../Caracal"), and add - ```../caMicroscope:/src/camicroscope``` to the volumes as shown below.
+
+![developFile](https://user-images.githubusercontent.com/40331239/113511371-158dd780-957d-11eb-8a26-c0f6a3554e72.jpg)
```

Comment:
I don't think we need to show this, we should just be able to say that it's a volume under the appropriate service.

Reply:
Again, to say that it's a volume under the appropriate service, it could be understood by people who are already aware of docker and yml files. But what I kept in mind while writing this document was that even a begineer who is obviously good at HTML, CSS and JS should be able to contribute to caMicroscope.